src/appendices/sql-install.rst

```diff
@@ -0,0 +1,63 @@
+Installing SQL
+==============
+
+.. index:: SQL
+
+A couple of chapters in this book cover SQL.
+Before we get started learning the ins and outs of the language, let's make sure we have everything properly installed!
+For this class, we will be using MySQL version 8.0.18, the most current version. Windows and Mac installations look a little different, so make sure you are following the appropriate directions for your machine.
+
+.. admonition:: Note
+
+   No matter your operating system, before you download any MySQL product, you are prompted to create an account.
+   You do not have to do so if you do not want to! Below the buttons to log in, there is an option to skip creating an account and proceed with the download.
+
+
+Mac Users
+---------
+
+First, we need to install MySQL itself.
+The `Community Server 8.0.18 download page <https://dev.mysql.com/downloads/mysql/>`_ has a dropdown menu for the operating system you need.
+Make sure you have selected MacOS before clicking *Download*!
+You want to download the version called "macOS 10.14 (x86, 64-bit), DMG Archive" that specifies it is the "(mysql-8.0.18-macos10.14-x86_64.dmg)". This option should also be at least 300 MB.
+
+#. When MySQL is done downloading, double-click on the disk image to launch the installer.
+#. Click through the windows to install MySQL and agree to the terms and conditions.
+#. When you reach the configuration window, you may have to enter a password for the root user. Pick a good password that you can remember.
+
+   .. admonition:: Warning
+
+      Once you create a password for the root user, you cannot change it! Be careful when typing out the password.
+
+Now that we have MySQL installed, we want to use it!
+You can setup MySQL for use in the terminal, however, many don't actually like it.
+When working with MySQL in the terminal, all commands are final and you can't use any fun tools to visualize the tables.
+Sounds like we should use a desktop client at first to practice! For these reasons, we will be using MySQL Workbench as our GUI client to work with our databases.
+
+The `MySQL Workbench 8.0.18 download page <https://dev.mysql.com/downloads/workbench/>`_ has a dropdown menu for the operating system. Make sure MacOS is selected before downloading.
+
+Once you have MySQL workbench installed, double-click on the disk image to launch the installer.
```

Comment:
"...installed, ...installer"?

Maybe the first part could be, "Once you have the MySQL package..."?

Reply:
Done! Thanks!